lib/systems/doubles.nix

```diff
@@ -27,7 +27,9 @@ let
     # Linux
     "aarch64-linux" "armv5tel-linux" "armv6l-linux" "armv7a-linux"
     "armv7l-linux" "i686-linux" "loongarch64-linux" "m68k-linux" "microblaze-linux"
-    "microblazeel-linux" "mipsel-linux" "mips64el-linux" "powerpc64-linux"
+    "microblazeel-linux" "mips-linux" "mips64-linux" "mips64el-linux"
+    "mipsel-linux" "mipsisa32r6-linux" "mipsisa32r6el-linux"
+    "mipsisa64r6-linux" "mipsisa64r6el-linux" "powerpc64-linux"
```

Comment:
We could do the same here and group arm, mipsel, power/s390, riscv and others or so but it is really just formatting which is not really important.